llvm/include/llvm/MC/MCDirectives.h

```diff
@@ -48,6 +48,10 @@ enum MCSymbolAttr {
   MCSA_WeakDefAutoPrivate,      ///< .weak_def_can_be_hidden (MachO)
   MCSA_WeakAntiDep,             ///< .weak_anti_dep (COFF)
   MCSA_Memtag,                  ///< .memtag (ELF)
+  MCSA_Code,                    ///< symbol is code (GOFF)
+  MCSA_Data,                    ///< symbol is data (GOFF)
```

Comment:
The above two duplicate the semantics of `MCSA_ELF_TypeFunction` and `MCSA_ELF_TypeObject` in a sense.  I'm wondering if we can hook into that (already existing) common code logic ...